evap/evaluation/models.py

```diff
@@ -1275,16 +1269,15 @@ def is_student(self):
 
     @property
     def is_contributor(self):
-        return self.contributions.exists()
+        return self.contributions.exists() or self.is_responsible
```

Comment:
Open discussion: Do we want this or do we want the definition like "Contributors are all the persons that have contributions".

Reply:
we want an exact definition. so i'll change that.